### DISCUSSION TITLE
Display shows only splash screen

### DISCUSSION THREAD

**Discussion by StephaneThiebaut** [Desuuuu/klipper#129] [Q&A]
Hi,
I tried to install Klipper for my Ender 6. I selected the following options:
![image](https://user-images.githubusercontent.com/21969436/196026378-d90048da-cdb3-4013-9dae-7a601a0f3823.png)

I've got an error: Conflicting definition for constant 'RESERVE_PINS_serial'
I can see in compile_time_request.txt that RESERVE_PINS_serial is declared twice.
I modified src/stm32/t5uid1/serial.c: I replaced all instances RESERVE_PINS_serial with RESERVE_PINS_t5uid1. It compiles now with no error. So I flashed the printer. I flashed the display and now it only shows klipper logo only...
I've connected a RPi with OctoPrint and it works, but I can't use the display. Did I make a mistake?
Thanks for your help.

**Comment by Desuuuu** [comment]
Thanks for reporting the compilation issue, it should be fixed now.

Your problem seems to indicate a communication issue between your mainboard and screen. You should verify that you have selected the right communication interface for your specific board and that your wiring is correct.

**Reply by StephaneThiebaut** [reply]
Thank you very much,
In fact I didn't have the good config file in raspberry, there were no t5uid1 section, and I didn't have understood that the display needed the raspberry to work. Now everything is OK.
Have a good day.
